frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java

```diff
@@ -115,19 +115,19 @@ private void initModelStore() {
 
         ModelManager modelManager = ModelManager.getInstance();
         int workers = configManager.getDefaultWorkers();
-        if ("ALL".equalsIgnoreCase(loadModels)) {
-            String modelStore = configManager.getModelStore();
-            if (modelStore == null) {
-                logger.warn("Model store is not configured.");
-                return;
-            }
+        String modelStore = configManager.getModelStore();
 
-            File modelStoreDir = new File(modelStore);
-            if (!modelStoreDir.exists()) {
-                logger.warn("Model store path is not found: {}", modelStore);
-                return;
-            }
+        if (modelStore == null) {
```

Comment:
Out of this PR's scope, but I wonder if we should actually throw an exception in this case as the modelStore is being relied on in several other places. It might make sense to crash MMS in case the store isn't set, rather than simply warning against it and aborting the model registration.